network-api/networkapi/wagtailpages/pagemodels/buyersguide/utils.py

```diff
@@ -41,3 +41,10 @@ def sort_average(products):
     `products` is a QuerySet of ProductPages.
     """
     return sorted(products, key=lambda p: p.creepiness)
+
+
+def get_featured_cta(self):
```

Comment:
**Question** Why is this parameter called `self` outside of a class?

Reply:
Great catch Tibor, I think that this could be confusing to read with `self`, so I updated it to `page`, as we are passing in a page, finding the home page in it's list of ancestors, and then returning the CTA if applicable.